helper/validation/validation_test.go

```diff
@@ -427,36 +427,7 @@ func TestValidationNoZeroValues(t *testing.T) {
 	})
 }
 
-func runTestCases(t *testing.T, cases []testCase) {
-	matchErr := func(errs []error, r *regexp.Regexp) bool {
-		// err must match one provided
-		for _, err := range errs {
-			if r.MatchString(err.Error()) {
-				return true
-			}
-		}
-
-		return false
-	}
-
-	for i, tc := range cases {
-		_, errs := tc.f(tc.val, "test_property")
-
-		if len(errs) == 0 && tc.expectedErr == nil {
-			continue
-		}
-
-		if len(errs) != 0 && tc.expectedErr == nil {
-			t.Fatalf("expected test case %d to produce no errors, got %v", i, errs)
-		}
-
-		if !matchErr(errs, tc.expectedErr) {
-			t.Fatalf("expected test case %d to produce error matching \"%s\", got %v", i, tc.expectedErr, errs)
-		}
-	}
-}
-
-func TestFloatBetween(t *testing.T) {
+func TestValidateFloatBetween(t *testing.T) {
```

Comment:
lol it seems we've been a bit inconsistent, the integer equivalents are names `TestValidationIntAtMost`.